_posts/2023-07-19-Remilia-interview.md

```diff
@@ -2,6 +2,7 @@
 title: "Interview with contributor Remilia Scarlet"
 author: beta-ziliani,Remilia
 summary: "Music playback projects in Crystal, games engines, and a bit of Lisp"
+category: community
```

Comment:
what's this, why just this one?

Reply:
Oh, that's an unintended change. Thanks for noticing 🙇 

(I have a different branch with post categories, apparently this spilled over)